conda.recipe/meta.yaml

```diff
@@ -5,16 +5,23 @@ package:
 source:
   path: ../
 
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
+
```

Comment:
OK, now commenting on this:

>The rasterize stuff is for backend pdf output, which i was working on when
i started working the anaconda build stuff in earnest. The source there is
one of the freebies they give you with phantom, and is a good starting
point.

OK

>i currently install phantom with npm... But it has literally nothing to do
with node, so hopefully that can be moved to a conda dependency... 
Like i said, phantomjs2 is a high value thing to have packaged for conda
users... We're playing hurt until then. 

OK, new need to make it happen or ask for help if we can not build it...

>At present, you have to run the
notebook server with an awful websockets flag too get it to work with old
phantom. 

Yep...

>Also, a lot of things look weird, and nobody cares, because most
of the WebKit  issues have been fixed in the last 4 years. But the project
(which has way more users than developers, if you know what i mean) has
been unable to get a stable Linux build of 2 out. Further, some of the
outstanding issues are directly related to pdf! So for now, I'm
concentrating on best-effort results with phantom 1.9!

OK

>Anyhow, phantom will be used at build time to run tests/update docs(?), but
it will eventually be a shipped feature that an end user would activate
from either the notebook, the dashboard, or the command line. This may
actually ship as a plugin... And certainly wouldn't show up in the pypi
build. But again, for the time being, i will continue to hoot and wave this
femur at the monolith.

Great, thanks for this work...

>My intent is to make a pdf exporter that:
- exports to zip (as the current one does, probably will just subclass)
- extract, server the zip contents
  - will probably need access to the running notebook server for notebook
local images, etc...
- open it with phantom
- advance through each state, for each slide
  - make a numbered PDF
- join the PDFs (in python)

Sounds good

>With my layout approach (moving content, to potentially multiple places in
the deck), this is the only way to get everything to appear on the page...
Too bad! However, page-at-a-time will give us certain freedoms, such as
different sized pages (to appease people that want scrolled content), and
being able to retemplate things like page numbers without relying on css's
awful support.

>Widgetful pdf generation, which presently will require a kernel in the full
notebook server, is a distant stretch goal... It's not worth doing until we
have phantomjs2... 

I agree...

>And will carry lots of assumptions, such as that it gets
a new kernel, and runs all the cells.